src/styles/ThemeStylesContext.ts

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import styles from './styles';
 
-const ThemeStylesContext = React.createContext(styles);
+// TODO: Change "unknown" once "styles.js" is typed
+const ThemeStylesContext = React.createContext<unknown>(styles);
```

Comment:
There is a [PR](https://github.com/Expensify/App/pull/27660) that migrates styles.ts to Typescript, I think it would make sense to change the base branch of this PR to it. 

We had to migrate some of themes files to TS, so you will get some conflicts (it was typed differently)

Reply:
Done 👍